docs/docs/security.md

```diff
@@ -96,21 +90,23 @@ Here is a non-exhaustive set of examples:
 
 * **Message sizes**. If an enclave is known to process only two kinds of message of around 1 kilobyte or 100 
   kilobytes, then the size of the encrypted message by itself leaks information about what kind of message it is.
+  Conclave can pad all messages to make all encrypted messages appear the same size. This approach works
+  if message sizes don't vary wildly, and you can afford to use the bandwidth and storage required to set all messages
+  to their maximum possible size.
+
 * **Message processing time**. If an enclave processes a message type in 1 millisecond and another message type in 
   100 milliseconds, observing how much work the enclave does for a new message, can reveal what kind of message it is.
+
 * **Storage access patterns**. Suppose an enclave doesn't access the database when processing a message of type A, 
   but does when processing a message of type B, or accesses the database with a different sequence, number or type 
   of accesses. In that case, the host can learn the message type by observing those accesses.
+  Conclave's [persistent file system](persistence.md#PersistentEncryptedFilesystem) randomizes sector accesses to
```

Comment:
ditto. I believe this should be `#persistent-encrypted-filesystem`.

Reply:
Corrected.